assets/less/admin.less

```
@@ -1582,12 +1582,23 @@ body.wpuf-modal-open {
+        width: 1px;
+        height: 1px;
+        margin: 0;
+        padding: 0;
+        border: 0;
         opacity: 0;
-        width: 0;
-        height: 0;
         pointer-events: none;
+        -webkit-appearance: none;
+        appearance: none;
+        clip: rect(0 0 0 0);
```

Stylelint (17.13.0)

[error] Deprecated property "clip" (property-no-deprecated)

(property-no-deprecated)